### DISCUSSION TITLE
/playtimetop command

### DISCUSSION THREAD

**Discussion by Chris6ix** [EssentialsX/Essentials#4940] [Ideas and Feature Suggestions]
It would be really great to have a command like `/playtimetop` that works just like `/balancetop` but for playtimes :)
It's already possible to see each player's playtime with `/playtime [player]`, so having the top ones in a list with one command would be nice.

**Comment by Chew** [comment]
I'm fairly certain this was rejected due to limitations (such as impossible to get the playtime of offline players prior to 1.15) and was better left for a separate plugin. I do like the idea though.

**Reply by Chris6ix** [reply]
I didnt know that...it could be implemented for 1.15+ then

**Reply by datatags** [reply]
> I'm fairly certain this was rejected due to limitations (such as impossible to get the playtime of offline players prior to 1.15) and was better left for a separate plugin. I do like the idea though.

Is it something that a PR would be accepted for then or is it still "left for a separate plugin"? `/playtime` doesn't work for offline players below 1.15.2 anyway, and it looks like 87.8% of EssentialsX servers use 1.15.2 or higher already. I'd take a stab at a PR for it if it'd be looked at.
